File: Data Structures/stacks.c
#include<stdio.h>
void pop(int a[], int *top);
void push(int value, int a[], int *top);
#define MAX 100


   int main(int argc, char const *argv[])
{
  
    int n,x,a[MAX],top,value;
    printf("Enter the number of elements to be Entered\t");
    scanf("%d",&n);
    for (int i = 0; i < n; ++i)
    {
      scanf("%d", &a[i]);
      top = i;
    }
    printf("Enter the value to push\n");
    scanf("%d",&value);
    push(value,a,&top);
    pop(a,&top);
}

  void push(int value, int a[], int *top){
    if((*top)==MAX-1)
      printf("The stack is Full.\n");
    else{
      (*top) +=1;
      a[(*top)] = value;
      printf("The value %d was pushed\n", value);
    }
  }
  
void pop(int a[], int *top){
  if(*(top)==-1)
    printf("The Stack is empty\n");
  else{
    (*top) -= 1; 
  }
}

File: Data Structures/stacks.py
def push(data):
    N.append(data)
def pop(data):
    N.pop()
def peek(data):
    return data[-1]
def isempty(data):
    if(len(data)==0):
        return True
    else:
        return False
def isfull(data):
    if(len(data)==N):
        return True
    else:
        return False
stack = []
N = int(input("Enter the max number of items in the stack: "))
for i in range(N-3):
    x = input("Enter data : ")
    stack.append(x)
print(stack)